addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -487,7 +487,7 @@ registerWebsitePreviewTour("website_form_editor_tour", {
         trigger: '.dropdown:has([name="link_style_color"]) > button',
         run: "click",
     }, {
-        trigger: "[data-value=secondary]",
+        trigger: "[data-value=custom]",
```

Comment:
not sure about this. i guess i need some context to understand

Reply:
Further steps from the tour use options that are only available for "Custom" buttons nowadays.
This step fails in master because those options used to be available on any buttons - but now Primary and Secondary are defined once and for all inside the theme options tab.